UM/Settings/Models/ContainerPropertyProvider.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2017 Ultimaker B.V.
 # Uranium is released under the terms of the LGPLv3 or higher.
 
-from PyQt5.QtCore import QObject, QVariant, pyqtProperty, pyqtSignal
+from PyQt6.QtCore import QObject, QVariant, pyqtProperty, pyqtSignal
```

Comment:
```suggestion
from PyQt6.QtCore import QObject, pyqtProperty, pyqtSignal
```
Remove unused import